packages/fuselage-hooks/src/useLocalStorage.server.spec.ts

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Why we're using this `.server` tests for this kind of functionality only present on the client side?

Reply:
ssr

Reply:
Yeah, SSR. A React component using our hooks should be able to run on Node.js to generate static HTML, therefore we must guarantee that they don't break when web APIs are missing.

Reply:
I discuss it deeper with gazzo, thanks for the explanation guys!